app/frontend/src/cljs/teet/asset/materials_and_products_view.cljs

```diff
@@ -50,11 +53,29 @@
     :parameter (format-properties atl value)
     (str value)))
 
+(defn- table-section-header [e! query listing-opts closed-set {ident :db/ident :as header-type} subtotal]
+  [table/listing-table-body-component listing-opts
+   [container/collapsible-container-heading
+    {:container-class [(<class common-styles/flex-row)
+                       (when (= "fclass" (namespace ident))
+                         (<class common-styles/indent-rem 1))]
+     :open? (not (closed-set ident))
+     :on-toggle (e! cost-items-controller/->ToggleOpenTotals ident)}
+    [:<>
+     [url/Link {:page :materials-and-products
+                :query (merge query {:filter (str ident)})}
+      (asset-ui/label header-type)]
+     (when subtotal
+       [:div {:style {:float :right :font-weight 700
+                      :font-size "80%"}} subtotal])]]])
+
```

Comment:
It would be pretty easy to unify this with the one in cost-groups-view and move it to asset-ui.